e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -736,6 +736,8 @@ public async Task DPS_Registration_AmqpWs_X509_IndividualEnrollment_InvalidGloba
                     : _idPrefix + AttestationTypeToString(attestationType) + "-" + Guid.NewGuid();
             }
 
+            bool shouldCleanupEnrollment = groupId == null || groupId == TestConfiguration.Provisioning.X509GroupEnrollmentName;
```

Comment:
It shouldn't equal the hardcoded name if we should cleanup, right?

```suggestion
            bool shouldCleanupEnrollment = groupId == null || groupId != TestConfiguration.Provisioning.X509GroupEnrollmentName;
```

Reply:
This is actually a problem in my changes. I don't know why your PR has my changes in it too.

Reply:
Either way, you can fix this for me, please.

Reply:
On it